packages/cli/src/commands/test.js

```diff
@@ -24,51 +50,45 @@ export const builder = (yargs) => {
     )
 }
 
-export const handler = async ({ side }) => {
-  const { base: BASE_DIR, cache: CACHE_DIR } = getPaths()
+export const handler = async ({ side, watch, watchAll, collectCoverage }) => {
+  const { cache: CACHE_DIR } = getPaths()
+  const sides = [].concat(side).filter(Boolean)
 
-  const DB_URL = process.env.TEST_DATABASE_URL || `file:${CACHE_DIR}/test.db`
+  const args = [
+    '--passWithNoTests',
+    watch && '--watch',
+    collectCoverage && '--collectCoverage',
+    watchAll && '--watchAll',
+  ].filter(Boolean)
 
-  const execCommands = {
-    api: {
-      cwd: `${BASE_DIR}/api`,
-      cmd: `DATABASE_URL=${DB_URL} yarn rw db up && yarn jest`,
-      args: [
-        '--passWithNoTests',
-        '--config ../node_modules/@redwoodjs/core/config/jest.config.api.js',
-      ],
-    },
-    web: {
-      cwd: `${BASE_DIR}/web`,
-      cmd: 'yarn jest',
-      args: [
-        '--passWithNoTests',
-        '--config ../node_modules/@redwoodjs/core/config/jest.config.web.js',
-      ],
-    },
+  // Watch unless on CI or explicitly running all tests
+  if (!process.env.CI && !watchAll && !collectCoverage) {
+    // https://github.com/facebook/create-react-app/issues/5210
+    const hasSourceControl = isInGitRepository() || isInMercurialRepository()
+    args.push(hasSourceControl ? '--watch' : '--watchAll')
   }
 
-  const tasks = new Listr(
-    side.map((sideName) => {
-      const { cmd, args, cwd } = execCommands[sideName]
-      return {
-        title: `Running '${sideName}' jest tests`,
-        task: () => {
-          return execa(cmd, args, {
-            stdio: 'inherit',
-            shell: true,
-            cwd,
-          })
-        },
-      }
-    }),
-    {
-      renderer: VerboseRenderer,
-    }
-  )
+  args.push('--config', './node_modules/@redwoodjs/core/config/jest.config.js')
```

Comment:
I think we should use `getPaths().base` and use `path.join()` so that it works on windows.

I thought we're going to use `getConfig()`? and resort to the locally available jest configuration?

Reply:
Good one!

That's an option. This config is the umbrella config that searches for project configs inside your app. We could either:
1. Keep this config inside RW. It's unlikely that you'll need to extend this one. (CMIIW)
2. Locate it in the root of the app.